copier/main.py

```diff
@@ -662,21 +681,23 @@ def run_update(self) -> None:
             print(
                 f"Updating to template version {self.template.version}", file=sys.stderr
             )
+        self.apply_update()
+
+    def apply_update(self):
```

Comment:
It seems to me like this one method should be private, as it doesn't make sense to call it directly from API, right?

If so:

```suggestion
    def _apply_update(self):
```

Reply:
Done!